src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -22,18 +22,20 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
     ============================================================
-                     _ILLink
+                     ILLink
 
     Replace the files to be published with versions that have been
     passed through the linker. Also prevent removed files from being
     included in the generated deps.json.
     ============================================================
     -->
-  <Target Name="_ILLink"
+  <Target Name="ILLink"
           Condition=" '$(PublishTrimmed)' == 'true' And
                       '$(_TargetFrameworkVersionWithoutV)' >= '3.0' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' "
           DependsOnTargets="_RunILLink">
 
+    <NETSdkWarning ResourceName="ILLinkWarning" />
```

Comment:
I wonder if this should be information. It doesn't seems like a good experience that if you add PublishTrimmed to a project where you use msbuild's warnaserror, then you immediately get a failed build.

Reply:
Thinking about how to do this... is it fine to use `<Message Text="..." Importance="High" />`, or do I need to worry about localization? Would you like me to introduce a `NETSdkInformation`?

Reply:
I added `NETSdkInformation`.